agent/src/main/java/com/criteo/hadoop/garmadon/agent/tracers/hadoop/hdfs/FileSystemTracer.java

```diff
@@ -314,24 +314,27 @@ private static Object callDistributedFileSystem(@SuperCall Callable<?> zuper, @T
 
     private static Object executeMethod(@SuperCall Callable<?> zuper, String uri, String src, String dst, String fsAction, String username) throws Exception {
         long startTime = System.nanoTime();
+        DataAccessEventProtos.FsEvent.Status status = DataAccessEventProtos.FsEvent.Status.SUCCESS;
         try {
             return zuper.call();
         } catch (Exception e) {
+            status = DataAccessEventProtos.FsEvent.Status.FAILURE;
```

Comment:
As discussed not all hdfs query throw exception in case of failure but a boolean

Reply:
Done

Reply:
Done